plugin/src/steps/source-nodes/update-nodes/content-update-interval.js

```diff
@@ -69,22 +124,17 @@ const refetcher = async (
  * Starts constantly refetching the latest WordPress changes
  * so we can update Gatsby nodes when data changes
  */
-const startPollingForContentUpdates = (helpers, pluginOptions) => {
+const startPollingForContentUpdates = (helpers) => {
   if (
     process.env.WP_DISABLE_POLLING ||
     process.env.ENABLE_GATSBY_REFRESH_ENDPOINT
   ) {
     return
   }
 
-  const { verbose } = store.getState().gatsbyApi.pluginOptions
+  const { verbose, develop } = store.getState().gatsbyApi.pluginOptions
 
-  const msRefetchInterval =
-    pluginOptions &&
-    pluginOptions.develop &&
-    pluginOptions.develop.nodeUpdateInterval
-      ? pluginOptions.develop.nodeUpdateInterval
-      : 300
+  const msRefetchInterval = develop?.nodeUpdateInterval ?? 300
```

Comment:
Should this value default to 5000 like in the other locations? 

Nit: Is there a way to re-use that value so the default doesn't need to be updated in multiple places?

Reply:
Ahh good point :) actually it doesn't need to have a fallback here as the redux store has a default already. I'll remove this entirely

Reply:
And we could use a constant to tie it all together 🤔 we should probably do that for all of the default values actually. Maybe we should do that in a separate PR